docs/iceberg-rest-service.md

```diff
@@ -147,6 +147,26 @@ If you have a JDBC Iceberg catalog prior, you must set `catalog-backend-name` to
 You must download the corresponding JDBC driver to the `iceberg-rest-server/libs` directory.
 :::
 
+#### Multi catalog support
+| Configuration item                               | Description                                                                                                                  | Default value                                                                   | Required | Since Version |
+|--------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|----------|---------------|
+| `gravitino.iceberg-rest.catalog-provider-impl`   | The implementation of IcebergTableOpsProvider defines how the Iceberg REST catalog server gets iceberg catalogs.             | `org.apache.gravitino.iceberg.common.ops.ConfigBasedIcebergTableOpsProvider`    | No       | 0.6.0         |
+
+You can also specify catalog parameters by setting configuration entries in the style `gravitino.iceberg-rest.catalog.<catalog name>.<param name>=<value>`.
```

Comment:
From the perspective of a normal user:
1. what's the relationship normal configuration and multi catalog configurations, like  `gravitino.iceberg-rest.catalog.hive_backend.catalog-backend` and `gravitino.iceberg-rest.catalog-backend`?
2. if defined a catalog xxx, how to use it?